docs/usage/vc.md

```diff
@@ -1,7 +1,59 @@
-# Virtual Controlplane
+# Virtual Control Plane
+
+The `virtual-controlplane` helm chart is installing the `kube-apiserver` and `kube-controller-manager` which are the core
+building blocks of our virtual Kubernetes cluster.
+
+## Prerequisites
+
+* Nginx ingress controller is installed in your Kubernetes cluster
+* DNS is configured for your external Ingress load balancer
+
+## Configuration
+
+Before we begin with the installation it is recommended to override some of the default [values](https://github.com/onmetal/virtual-controlplane/blob/main/charts/virtual-controlplane/values.yaml).
+
+For tha we need to create a `values.yaml` file containing the following entries.
+
+```yaml
+apiServer:
+  externalDNSName: <external ingress DNS name>
+  tls:
+    staticToken: securePasswordToken
+```
+
+!!!note
+        The helm installation will run through without changing those values. This might be OK if you deploy it into a local Minikube or Kind cluster. However you won't be able to access your cluster since the `Ingress` won't be configured correctly. Also there is a huge security risk as the static token to can be guessed relatively easy exposing your API server to fraudulent access.
```

Comment:
```suggestion
        The helm installation will run through without changing those values. This might be OK if you deploy it into a local Minikube or Kind cluster. However you won't be able to access your cluster since the `Ingress` won't be configured correctly. Note, that there is a huge security risk as the static token can be guessed relatively easy exposing your API server to fraudulent access.
```